recipes/dsfmt/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
+make std
+make std-check
+make sse2-check
```

Comment:
No `make check` or `make test` step for all of these?

Reply:
`make check` just makes a copy of a bash script `check.sh` and makes it executable